Sheppard/build.gradle

```diff
@@ -32,13 +32,14 @@ android {
 
 repositories {
     maven {
-        url "https://raw.github.com/socialvibe/truex-tv-integrations/android"
+        url "https://ctv.truex.com/android/maven"
```

Comment:
maybe, as we add more folder on the ctv.truex.com, we should add a one-line readme file somewhere inside the folder explaining what this folder is for(purpose, and if it is for prod/qa), and who is responsible for the content of the folder.